constants/constants.go

```diff
@@ -2,3 +2,83 @@ package constants
 
 // CustomDateFormat is a date format used in the application to parse date string/object into a usable format
 const CustomDateFormat = "2006-01-02"
+
+// ActionItemDataType is datatype for ActionItems's
```

Comment:
Use just `Type` instead of `DataType`.
No need of `'s`.
Similarly for Action type below.